server/src/test/java/org/opensearch/action/RenamedTimeoutRequestParameterTests.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action;
+
+import org.junit.AfterClass;
+import org.opensearch.OpenSearchParseException;
+import org.opensearch.client.node.NodeClient;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.rest.action.cat.RestNodesAction;
+import org.opensearch.test.OpenSearchTestCase;
+import org.opensearch.test.rest.FakeRestRequest;
+import org.opensearch.threadpool.TestThreadPool;
+
+import static org.hamcrest.Matchers.containsString;
+
+/**
+ * As of 2.0, the request parameter 'master_timeout' in all applicable REST APIs is deprecated,
+ * and alternative parameter 'cluster_manager_timeout' is added.
+ * The tests are used to validate the behavior about the renamed request parameter.
+ * Remove the test after removing MASTER_ROLE and 'master_timeout'.
+ */
+public class RenamedTimeoutRequestParameterTests extends OpenSearchTestCase {
+    private static final TestThreadPool threadPool = new TestThreadPool(RenamedTimeoutRequestParameterTests.class.getName());
```

Comment:
Does this have to be static?

Reply:
Ah, because the method with `@AfterClass` annotation has to be `static`, so I also added `static` word to `threadPool` variable.

Reply:
> to define a constant, use static final

Yeah, the issue here is that a thread pool can't really be a "constant" because it is a stateful and mutable object (hence the need to terminate it). For any scalar values or immutable objects it would be fine to make them `static final` constants.

A potentially valid reason for making the thread pool static would be that if it was expensive to create and tear down, so you only wanted to do it once and run multiple tests with it. That carries the risk of tests interfering with one another since they would share mutable static state, so that should only be done in cases where it makes a big difference in test run time.

Reply:
> Ah, because the method with @AfterClass annotation has to be static

Just use the `@After` annotation on a non-static method and it will be run after every test case.

Reply:
Thank you so much for telling me the detail! 👍 👍 (String is immutable, so it's fine to be static 😄)
Sorry my mistake, I wanted to say the `threadPool` can be terminated after all test cases run and can be shared with test cases, because I think the interfering is not a problem, because my test cases only care about the request parameters, and doesn't check the result.
So looks fine to keep the threadpool it static.

Reply:
You're right that the current implementation will likely work fine with a static thread pool. But in absence of a strong reason to use shared static state, you should prefer to avoid shared static state. You never know what kind of test the next developer will add to this test class :)

Reply:
Ok, I updated the threadpool to be non-static, and terminate the threadpool after every test cases, hope it will not expensive to tear down. 😄 Thank you!